bot.py

```diff
@@ -130,18 +184,32 @@ async def help_command(ctx):
 
 
 @bot.command(name="rank", help="Displays the information about the summoner.")
-async def get_rank(ctx, *, name: str):  # using * for get a summoner name with space
+async def get_rank(ctx, name="--help"):  # set name attribute to default help command
     """Sends the summoner's rank information to the bot"""
     try:
         # typing indicator
         async with ctx.typing():
             await asyncio.sleep(1)
 
+        # displays the get_rank's help command
+        if name == "--help":
+            create_help_title_desc(bot, f"{get_rank.name}", True)
+
+            # pylint: disable=line-too-long
+            create_help_fields(
+                [
+                    "This command displays the information about the summoner.",
+                    "The information includes the summoner's name, rank, total games played and the win rate.",
+                    f"__NOTE__:   **{get_rank.name}** command only accepts one summoner name.",
+                ]
+            )
+
+            return await ctx.send(embed=create_help_embed())
```

Comment:
You are returning `await ctx.send(embed=create_help_embed())` why? what's the output of  `await ctx.send(embed=create_help_embed())`?

Reply:
I attached the output picture in `Overview.
Also `create_help_embed()` will return `help_embed` from `help_commands.py`